### PR TITLE
Supplement NIC prerequisites for installation from Docker

### PR DESCRIPTION
**Pull request by leoyjchang**
The original RackHD installation from Docker guide missed part of NIC prerequisites which I think should be added.

@linlynn @lanchongyizu

### CODE REVIEW

File: docs/run_rackhd/docker_based_installation.rst [hunk 1]
```diff
@@ -3,8 +3,63 @@ Installation from Docker
...
+            post-up ifconfig eth1 promisc
+
+    .. tab:: Ubuntu 16.04
+
+        Start with an Ubuntu xenial(16.04) instance with 2 nics:
+
+        * ``ens160`` for the ``public`` network - providing access to RackHD APIs, and providing
+          routed (layer3) access to out of band network for machines under management
+
+        * ``ens192`` for dhcp/pxe to boot/configure the machines
```
Comment (iceiilin) [inline docs/run_rackhd/docker_based_installation.rst]: ens160 and ens192 might not be fixed as it's related to NIC's physical positions in the pci system after Ubuntu 15.04.

Comment (leoyjchang) [reply docs/run_rackhd/docker_based_installation.rst]: agreed, let me add notes for the nic name.

Comment (leoyjchang) [reply docs/run_rackhd/docker_based_installation.rst]: @iceiilin , added additional notes for ethernet name, please review again, thanks~~

Comment (lanchongyizu) [reply docs/run_rackhd/docker_based_installation.rst]: Same problem with eth0 and eth1. Here eth0/eth1 or ens160/ens192 are just examples.

**Pull request merged**